Plugins/NativeInput/Source/Android/NativeInputAndroid.cpp

```diff
@@ -0,0 +1,9 @@
+#include "..\Shared\NativeInput.h"
+
+namespace Babylon::Plugins
+{
+    bool NativeInput::Impl::HasMouse()
+    {
+        return false;
```

Comment:
nit: This is probably fine for now, but I don't think this is a safe assumption going forward, is it? Android devices do support mice, I think.

Reply:
I'll add todos to these currently returning false